README.md

```diff
@@ -60,15 +60,26 @@ docker-machine create -d ovh node-1
 |``--ovh-consumer-key`` or ``$OVH_CONSUMER_KEY``            |Consumer Key      |none      |yes|
 |``--ovh-region``                                           |Cloud region      |GRA1      |no|
 |``--ovh-flavor``                                           |Cloud Machine type|vps-ssd-1 |no|
+|``--ovh-image``                                            |Cloud Machine image|Ubuntu 16.04 |no|
+|``--ovh-ssh-user``                                         |Cloud Machine SSH User|ubuntu |no|
 |``--ovh-project``                                          |Cloud Project name|single one|only if multiple projects|
 
-Note: OVH credentials may be supplied through arguments, environment or configuration file, by order
+Example with parameters:
```

Comment:
Can you add a simple example first and then insert this new example as "advanced" ? Maybe be structure like

```
*Basic example*:

...

*Advanced example: Use CoreOS*

...

```